src/client/cli/cmd/launch.cpp

```diff
@@ -289,6 +320,17 @@ mp::ParseCode cmd::Launch::parse_args(mp::ArgParser* parser)
         request.set_disk_space(arg_disk_size);
     }
 
+    if (parser->isSet(mountOption))
+    {
+        for (const auto& value : parser->values(mountOption))
+        {
+            auto mount_values = value.split(':');
+            auto mount_source = mount_values.first();
+            auto mount_target = mount_values.size() == 1 ? mount_source : mount_values.at(1);
+            mount_routes.insert({mount_target, mount_source});
+        }
+    }
```

Comment:
Would it be possible to write a/some test(s) for this block of code?

Reply:
yes. i'm currently in the process of writing tests for this feature :smile:

Reply:
Awesome, thanks!